src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -523,5 +523,35 @@ public override async Task<VSInternalWrapWithTagResponse> RazorWrapWithTagAsync(
 
             return response;
         }
+
+        public override async Task<VSInternalInlineCompletionList?> ProvideInlineCompletionAsync(VSInternalInlineCompletionRequest inlineCompletionParams, CancellationToken cancellationToken)
```

Comment:
Hmmm we'll want to wrap `VSInternalInlineCompletionRequest` in our own type so we can pass a language kind or we'll want to rename this method to `ProvideCSharpInlineCompletionAsync`. Reason being that WebTools / HTML also have some snippet expansions I can imagine them wanting to light up.

/cc @ToddGrun @jimmylewis